src/modules/iasql_functions/iasql.ts

```diff
@@ -1766,7 +1771,7 @@ async function recreateRelation(
     .map(or => ({
       targetEntity: or.inverseEntityMetadata.target,
       propertyName: or.propertyName,
-      colsWithReferences: or.joinColumns.map(jc => [jc.databaseName, jc.referencedColumn?.databaseName]),
+      colsWithReferences: or.joinColumns.map(jc => [jc.propertyName, jc.referencedColumn?.propertyName]),
```

Comment:
are you sure about this change?? For an entity property called `myProp`, `.databaseName` would give us `my_prop`... are we sure `.propertyName ` returns this?

Reply:
Yes, it fails hard without this. TypeORM is internally translating this back to the `databaseName` when we feed it into `findOne` below -- the `find`/`findOne` queries assume the `propertyName` formatting because it's an ORM trying to hide these database details from you.

I am surprised it worked before.